src/core_modules/capture-core-utils/date/index.js

```diff
@@ -0,0 +1,2 @@
+// @flow
+export { getFormattedStringFromMomentUsingEuropeanGlyphs } from './date.utils';
```

Comment:
wanna add this empty line in the end? Eslint complains when we dont have an empty line in the end of our js files.
```suggestion
export { getFormattedStringFromMomentUsingEuropeanGlyphs } from './date.utils';

```

Reply:
👍 I believe my eslint wasn't up and running when I committed this.